widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/EngagementStates.kt

```diff
@@ -47,6 +47,28 @@ internal fun ChatEngagementStates(
     }
 }
 
+/**
+ * Default theme for Call Engagement states
+ */
+internal fun CallEngagementStates(
+    pallet: ColorPallet
+): EngagementStatesTheme? {
+    val operatorTheme = OperatorDefaultTheme(pallet)
+
+    return pallet.run {
+        composeIfAtLeastOneNotNull(operatorTheme, baseLightColorTheme) {
+            val engagementState =
+                EngagementState(title = baseLightColorTheme, description = baseLightColorTheme)
+            EngagementStatesTheme(
+                operator = operatorTheme,
+                connecting = engagementState,
+                connected = engagementState,
+                onHold = engagementState
```

Comment:
I'm not sure, but maybe it makes sense to rename `EngagementState` to `EngagementStateTheme`? It looks a bit confusing when it's named "state".